test/hive-connect.hcl

```
@@ -164,6 +186,28 @@ EOH
       mode = "bridge"
     }
 
+    task "waitfor-hive-database" {
```

```suggestion
    task "waitfor-hive-database" {
        restart {
            attempts = 5
            delay    = "15s"
        }    
```
same as above. 5 retries is enough for this one.